Code/Tools/ProjectManager/Source/CreateAGemScreen.h

```diff
@@ -34,36 +38,54 @@ namespace O3DE::ProjectManager
         explicit CreateGem(QWidget* parent = nullptr);
         ~CreateGem() = default;
 
+        void ClearFields();
+
+        ProjectManagerScreen GetScreenEnum() override
```

Comment:
Not sure why we haven't made this function const everywhere - but that'll touch a lot of files so we can do that in a separate review